modules/unsupported/flatgeobuf/src/test/java/org/geotools/data/flatgeobuf/FlatGeobufDataStoreFactoryTest.java

```diff
@@ -73,47 +76,51 @@ public void cantProcessWrongName() throws IOException {
         File file = temporaryFolder.newFile("points.shp");
         Map<String, Serializable> params = new HashMap<>();
         params.put("wrong name", file);
-        FlatgeobufDataStoreFactory dataStoreFactory = new FlatgeobufDataStoreFactory();
+        FlatGeobufDataStoreFactory dataStoreFactory = new FlatGeobufDataStoreFactory();
         assertFalse(dataStoreFactory.canProcess(params));
     }
 
     @Test
     public void createDataStoreFromFile() throws IOException {
         File file = temporaryFolder.newFile("points.fgb");
         Map<String, Serializable> params = new HashMap<>();
-        params.put("flatgeobuf-file", file);
```

Comment:
I am not sure it matters, but the Shapefile data store also uses "url" as its one and only parameter.  I used something more distinctive so I could tell whether someone was trying to load flatgeobuf just from the Map of params.

Reply:
Agreed might make sense but I don't want to stick out (fx. geojsonstore also have url)